examples/nemo/scripts/omp_gpu_trans.py

```diff
@@ -41,7 +41,8 @@
     enhance_tree_information, add_profiling
 
 from psyclone.psyGen import TransInfo
```

Comment:
Please could you clean up for pylint in this file - it doesn't like the ordering of imports and says `Schedule` is unused.

Reply:
Done